packages/libfabric/package.py

```diff
@@ -34,6 +34,8 @@ class Libfabric(BuiltinLibfabric):
     variant('cuda', default=False, description='Configure with cuda support')
     depends_on('cuda', when='+cuda')
 
+    variant('enable-cuda-dlopen', default=False, description='Use dlopen to open cuda library during runtime and load symbol from the library')
```

Comment:
The spack variant should probably be called `cuda-dlopen` (or maybe `cuda_dlopen`?  I don't know if dashes are Ok in spack variant names or not).  At any rate I think they usually omit the "enable" part when mapping variants to build arguments.